Story/Episode 6/st_062090.csv

```diff
@@ -0,0 +1,26 @@
+st_062090_0001#0,"""A.P.242/2/26/16:20"""
+st_062090_0010#0,"""What is...!"""
+st_062090_0020#0,"""You did it, Sara!"""
+st_062090_0030#0,"""Picking up dimensional transfer readings. Claris Claes, take them out."""
+st_062090_0040#0,"""Got it!"""
+st_062090_0050#0,"""They can precisely track our locations...? But that shouldn't be possible..."""
+st_062090_0060#0,"""That's just what happens when two Xion clones combine forces. Thanks for the real-time adaptation, Kamits."""
+st_062090_0070#0,"""〔You need not thank me〕<br>〔This is for the future after all〕"""
+st_062090_0080#0,"""Xiao!"""
+st_062090_0090#0,"""Kept you waiting, huh. You've done well making it here."""
+st_062090_0100#0,"""Sara?!"""
+st_062090_0110#0,"""Don't worry. She's only asleep."""
+st_062090_0120#0,"""It's thanks to her hard work that I'm able to be here as I am now."""
+st_062090_0130#0,"""Not rewarding all of that hard work... Would be rude."""
+st_062090_0140#0,"""He vanished?!"""
+st_062090_0150#0,"""...His readings are gone. He appears to have retreated."""
+st_062090_0160#0,"""Xiao... You made it back."""
+st_062090_0170#0,"""Just woken up, as someone put it. Well, I made it here in time because of that. You should be thanking Sara."""
+st_062090_0180#0,"""We heard a massive explosion not too long ago. Is the vessel still intact?"""
+st_062090_0190#0,"""Kashina's central core was destroyed. But the form a vessel takes is not what matters to the Dragons. Right?"""
+st_062090_0200#0,"""〔Correct〕〔What matters is the soul〕<br>〔Suitable vessels〕〔exist all around us〕"""
```

Comment:
``Correct`` needs a comma and change ``What matters is the soul`` to be lowercase and include a period. ``Exist all around us`` will need a period at the end as well.